WORKSPACE

```diff
@@ -8,6 +8,8 @@ load(
 git_repository(
     name = "bazel_skylib",
     commit = "df3c9e2735f02a7fe8cd80db4db00fec8e13d25f",  # `master` as of 2021-08-19
```

Comment:
Sorry for missing this - can't we just update to latest here instead of adding a patch? This issue has been fixed for more than three years.